static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -6,11 +6,24 @@ import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import once from 'lodash/once';
 import { prebid } from 'commercial/modules/prebid/prebid';
 
+const isGoogleWebPreview: () => boolean = () => {
+    try {
+        return navigator.userAgent.indexOf('Google Web Preview') > -1;
+    } catch (exception) {
+        return false;
+    }
+};
+
+if (!isGoogleWebPreview()) {
+    require('prebid.js/build/dist/prebid'); // eslint-disable-line global-require
+}
+
 export const setupPrebid: () => Promise<void> = once(() => {
     if (
         dfpEnv.externalDemand === 'prebid' &&
         commercialFeatures.dfpAdvertising &&
-        !commercialFeatures.adFree
+        !commercialFeatures.adFree &&
+        !isGoogleWebPreview()
```

Comment:
Perhaps add a test for this?  In case of regression as much as anything.

Reply:
No test as I imagined this itself is a test for checking to see if this is what is causing the underlying issue. Can add one.

Reply:
This is now tested, it had to be detached from `once` to do so.